apps/mi/mi-azure-functions/prod.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     labels:
       app.kubernetes.io/instance: mi-azure-functions-deployment
       app.kubernetes.io/name: mi-azure-functions-deployment
-    image: sdshmctspublic.azurecr.io/mi/azure-functions:prod-6326575-20220630110849
+    image: sdshmctspublic.azurecr.io/mi/azure-functions:prod-6326575-20220630110849 # {"$imagepolicy": "flux-system:mi-azure-functions-prod"}
```

Comment:
what is this comment for?

Reply:
Part of the Flux imagepolicy setup:
https://fluxcd.io/flux/guides/image-update/#configure-image-updates

Reply:
I did not realise this , I have to say that I don't like it